src/Form/GCSettingsForm.php

```diff
@@ -146,11 +146,24 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
         '#default_value' => $config->get('components.' . $id . '.up_next_title'),
       ];
 
-      $form['app_settings']['components'][$id]['empty_block_text'] = [
+      $form['app_settings']['components'][$id]['component']['empty_block_text'] = [
         '#type' => 'textfield',
         '#title' => $this->t('Text for empty block'),
         '#default_value' => $config->get('components.' . $id . '.empty_block_text'),
       ];
+
+      $select_options = [
+        'date_desc' => 'By date (New-Old)',
```

Comment:
For events in `date_desc` and `date_asc` we should have different label - `Event Date (desc)` and `Event Date (asc)`